src/components/logo/logo.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Heading,
+  VStack,
+  HStack,
+  Logo,
+  Link,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import appConfig from "appConfig";
+
+const DCLogo = () => {
+  return (
+    <>
+      <Link isUnderlined={false} href={`${appConfig.DC_URL}`}>
+        <HStack>
+          <Logo
+            pt="2"
+            pb="2"
+            name="nyplLionBlack"
+            size="xxsmall"
+            aria-label="NYPL Lion Logo"
+          />
+          <VStack
+            align="left"
+            sx={{
+              display: { sm: "none", md: "none", lg: "block" },
+            }}
+          >
+            <Heading
```

Comment:
Should I be passing this as an SVG instead?

Reply:
I didn't see this comment while I was adding my review, but I added a comment above that hopefully answers this question.